app/utils/createEntityReducer.js

```diff
@@ -173,16 +190,23 @@ export default function createEntityReducer({
     ...initialState
   };
 
-  const fetchTypes = isArray(fetch) ? fetch : [fetch];
+  const toArray = <T>(value: T | Array<T>) =>
```

Comment:
I think this function should be able to take the value `undefined` and return an empty array `[]`. That would remove the need to check and pass empty arrays when one of the types is undefined ( ` || [] ` ).

Reply:
Sounds good :+1: